### DISCUSSION TITLE
Starting Ghosts

### DISCUSSION THREAD

**Discussion by TheDyingYAK** [cmu-sei/GHOSTS#86] [Q&A] [answered]
This may be a simple question but how would I get Ghosts to start up and run as a specific user by itself without someone logging in to the machine and having ghosts start via a logon script? The most awesome option that I would want would be for it to start up without any external actions happening but I don't know what would have to go into it to actually make that happen? Any help or pointers/information would be greatly appreciated! Thanks!

**Comment by sei-dupdyke** [comment, marked as answer]
We typically do this as part of a powershell startup script that logs on a particular user, this is the function that starts the exe:

```powershell
function Start-Ghosts() {
	Set-Location -Path c:\step\ghosts
	Start-Process ghosts.exe -WindowStyle Hidden
}
```

We are working on a tool that we'll likely eventually OSS that has different agents log into different machines. This requires a management machine to send login commands, of course.

**Reply by TheDyingYAK** [reply]
ok thanks for the help. Thats what I have been doing and then I just had a script that would log onto random boxes via rdp but as you can guess that was a little noisy. I appreciate it.

**Reply by kchen1023** [reply]
Hi Dustin, are there any updates on the tool?

**Reply by sei-dupdyke** [reply]
Sorry, this effort slipped to the back of the stove — let me check on this and get back to you on OSS'ing it.
